### PR TITLE
docs: fix typos and improve French

### PR DESCRIPTION
**Pull request by xchopin**
L'écriture inclusive n'est pas du français. 

Source: l'Académie française.

### CODE REVIEW

File: content/_index.fr.md [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+## Le Manifeste Post-Méritocratie
 
 {{< translations >}}
 
 La méritocratie est un principe fondateur du mouvement Open Source, et l’idéal de la méritocratie est perpétué partout dans notre domaine dans la façon dont les gens sont recrutés, engagés, gardés et estimés.
 
 Mais la méritocratie a systématiquement montré qu’elle bénéficiait aux personnes qui ont des privilèges, et excluait celles qui sont sous-représentées dans la technologie. L’idée du mérite n’est en fait jamais clairement définie&nbsp;; au contraire, elle semble être une forme de reconnaissance que «&nbsp;cette personne est importante dans la mesure où elle est comme moi&nbsp;».
 
-(Si vous n’êtes pas familier·e avec les critiques de la méritocratie, réferrez-vous aux ressources de <a href="/meritocracy/">cette page</a>.)
+(Si vous n'êtes pas familier avec les critiques de la méritocracie, merci de vous référer aux ressources sur <a href="/meritocracy/">cette page</a> (anglais).)
```
Comment (guilhem) [inline content/_index.fr.md]: typo ~méritocracie~ méritocratie

**Comment by guilhem** [comment]
My 2 cent to maybe calm future debate:
@xchopin maybe you can change male / female order once over time

**Comment by xchopin** [comment]
> My 2 cent to maybe calm future debate:
> @xchopin maybe you can change male / female order once over time

On what line?

File: content/_index.fr.md (hunk 1 above)
Comment (xchopin) [reply content/_index.fr.md]: thanks for fixing that!

**Comment by xchopin** [comment]
> My 2 cent to maybe calm future debate:
> @xchopin maybe you can change male / female order once over time


a983598 done

**Comment by CoralineAda** [comment]
Thank you so much!

**Pull request merged**